cypress/components/pager/pager.cy.tsx

```diff
@@ -297,7 +299,7 @@ context("Test for Pager component", () => {
   });
 
   describe("check funtionality for Pager component", () => {
-    it.each([[-1], [-10], [-100], testData[0], testData[1]])(
+    it.each([-1, -10, -100, testData[0], testData[1]])(
```

Comment:
suggestion: you could spread the testData array here but it's not a big deal

```suggestion
    it.each([-1, -10, -100, ...testData)(
```